libraries/testbot/tests/testData/mainDialogTestCases.js

```diff
@@ -0,0 +1,61 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+let now = new Date();
+
+let tomorrow = formatDate(new Date().setDate(now.getDate() + 1));
+
+// Source: https://stackoverflow.com/questions/23593052/format-javascript-date-to-yyyy-mm-dd
+function formatDate(date) {
+    var d = new Date(date),
+        month = '' + (d.getMonth() + 1),
+        day = '' + d.getDate(),
+        year = d.getFullYear();
+
+    if (month.length < 2) month = '0' + month;
+    if (day.length < 2) day = '0' + day;
+
+    return [year, month, day].join('-');
+}
+
+module.exports = [
```

Comment:
MainDialog tests after refactoring can be found here: https://github.com/microsoft/BotBuilder-Samples/blob/8dcaafc51a73f6663c7f53d07f3c9eaec0b68d5d/samples/csharp_dotnetcore/13.core-bot.tests/Dialogs/MainDialogTests.cs

Reply:
Done.